FindSplash.cmake

```diff
@@ -96,7 +102,7 @@ if(Splash_ROOT_DIR)
     #
     find_library(Splash_LIBRARIES
       NAMES splash
-      PATHS $ENV{SPLASH_ROOT}/lib)
+      PATHS ${Splash_ROOT_DIR}/lib)
```

Comment:
that is very uncommon. Either call it `_ROOT` or `_DIR` but not both.
Furthermore, env/var hints are [deprecated](http://public.kitware.com/pipermail/cmake-developers/2014-June/022421.html) in CMake for the sake of the common `CMAKE_PREFIX_PATH` that is understood by all relevant functions (`find_path`, `find_library`, etc.).